repondeur/zam_repondeur/views/lectures.py

```diff
@@ -327,25 +327,31 @@ def get(self) -> Any:
 
     @view_config(request_method="POST")
     def post(self) -> Response:
-        amendements = self.get_amendements_from(self.request.POST)
+        # Special case: unbatch (TODO: move to a separate route)
```

Comment:
I think the checks must still be performed before unset-ing the batch?

Reply:
OK forget about this 😅 💤

Reply:
Well, we could check it's on the user's table to allow removing it from the batch